pkg/pgmodel/model/sql_test_utils.go

```diff
@@ -405,23 +405,23 @@ func (m *MockInserter) Close() {
 
 }
 
-func (m *MockInserter) InsertNewData(rows map[string][]SamplesInfo) (uint64, error) {
+func (m *MockInserter) InsertNewData(rows map[string][]Samples) (uint64, error) {
 	return m.InsertData(rows)
 }
 
 func (m *MockInserter) CompleteMetricCreation() error {
 	return nil
 }
 
-func (m *MockInserter) InsertData(rows map[string][]SamplesInfo) (uint64, error) {
+func (m *MockInserter) InsertData(rows map[string][]Samples) (uint64, error) {
 	for _, v := range rows {
 		for i, si := range v {
-			id, ok := m.InsertedSeries[si.Series.String()]
+			id, ok := m.InsertedSeries[si.GetSeries().String()]
```

Comment:
Micro op
```suggestion
                      seriesStr = si.GetSeries().String()
			id, ok := m.InsertedSeries[seriesStr]
```
and can be used in line 419 also (ui didn't allow me to apply the suggestion to both places)